addons/test_mail/tests/test_performance.py

```diff
@@ -19,13 +19,20 @@ def setUpClass(cls):
 
         # creating partners is required notably witn template usage
         cls.user_employee.write({'groups_id': [(4, cls.env.ref('base.group_partner_manager').id)]})
-        cls.user_test = cls.env['res.users'].with_context(cls._test_context).create({
+        cls.user_test = cls.user_test_inbox = cls.env['res.users'].with_context(cls._test_context).create({
             'name': 'Paulette Testouille',
             'login': 'paul',
             'email': 'user.test.paulette@example.com',
             'notification_type': 'inbox',
             'groups_id': [(6, 0, [cls.env.ref('base.group_user').id])],
         })
+        cls.user_test_email = cls.env['res.users'].with_context(cls._test_context).create({
+            'name': 'Georgette Testouille',
```

Comment:
TDE approves this naming.

Reply:
``Raoulette`` is probably better, to harmonize the data.

Reply:
I am really hesitant to really change to `Raoulette` to be consistent 😄